debian/gnome-software.install

```diff
@@ -13,4 +13,5 @@ usr/share/eos-discovery-feed/
 usr/share/glib-2.0/schemas/
 usr/share/gnome-shell/
 usr/share/man/
+usr/share/mime
```

Comment:
Nitpick: I think it’s conventional to end the line with `/` if you intend to install the whole directory, otherwise this could be mistaken for a file called `mime` in the `/usr/share` directory.

Reply:
Good point. Updated. Thanks.